### PR TITLE
Add archivematica-centos7 playbook

### PR DESCRIPTION
**Pull request by hakamine**

### CODE REVIEW

File: playbooks/archivematica-centos7/singlenode.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        name: "{{ item }}"
+        state: "started"
+        enabled: "yes"
+      with_items:
+        - "mariadb"
+        - "gearmand"
+      become: "yes"
+
+  roles:
+    - { role: "elastic.elasticsearch",
```
Comment (sevein) [inline playbooks/archivematica-centos7/singlenode.yml:40]: What are the brackets for?

**Comment by sevein** [comment]
This looks great @hakamine! I wouldn't hesitate to merge this and get some feedback from the community. Is this still experimental? Should we warn the users?

**Comment by hakamine** [comment]
The playbook is working fine in deploying AM and dependencies, but yesterday I realized that the official CentOS vagrant box doesn't have the VB guest additions which is an issue for development use (directory sync with the host doesn't work) ( refs. https://github.com/mitchellh/vagrant/issues/8374 , https://github.com/CentOS/sig-cloud-instance-build/issues/27 )...  found a different box with the guest additions included, will test and see how it goes.

**Comment by sevein** [comment]
You could as well mention in the docs that the environment is not suitable for development purposes - unless they install the guest additions manually. Not everything we share in `deploy-pub` needs to fit perfectly with the developer workflow IMO. It's up to you but I wouldn't say that's a priority.

File: playbooks/archivematica-centos7/singlenode.yml (hunk 1 above)
Comment (hakamine) [reply playbooks/archivematica-centos7/singlenode.yml:40]: I think brackets are needed when parameters ( variables ) are included (ref. http://docs.ansible.com/ansible/playbooks_roles.html#roles )

Comment (sevein) [reply playbooks/archivematica-centos7/singlenode.yml:40]: AFAIK that's just the syntax for inline mappings, see http://yaml.org/spec/1.0/#id2570829.

**Comment by hakamine** [comment]
Changed the vagrant box and now having synced folders working 😺 I think this is good to go now.

**Review by sevein — APPROVED** [review]
Glad to know that it worked! Thank you.

**Pull request merged**